addons/iot_drivers/websocket_client.py

```diff
@@ -82,6 +83,19 @@ def on_message(self, ws, messages):
                 case 'restart_odoo':
                     ws.close()
                     helpers.odoo_restart()
+                case 'remote_debug':
+                    if platform.system() == 'Windows':
+                        continue
+                    if not payload.get("status"):
+                        helpers.toggle_remote_connection(payload.get("token", ""))
```

Comment:
```suggestion
                    if not content.get("status"):
                        helpers.toggle_remote_connection(content.get("token", ""))
```
It wasn't working for me until I made this change